myloader.c

```diff
@@ -44,7 +44,9 @@
 guint commit_count = 1000;
 gchar *directory = NULL;
 gboolean overwrite_tables = FALSE;
+gboolean innodb_optimize_keys = FALSE;
```

Comment:
Any reason not to make this on by default? I can't think of any down sides.

Reply:
Not all the time it is better to use it. If you have small tables with just a few single column indexes, then you will see that there is an increase in the times which will no be more than 10%. This functionality is very useful when you have tables with millions of rows and a few of multi column indexes.